src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -120,10 +120,16 @@ export class HuiGraphHeaderFooter extends LitElement
         top: calc(50% - 28px);
         left: calc(50% - 12px);
       }
+      .info-container {
```

Comment:
`info-container` and `spinner-container` have the same styles, use the same class for both cases.